webserver/views/data.py

```diff
@@ -195,72 +195,85 @@ def get_tag(name):
         return {}
 
 
-def _interpret_high_level(hl):
+def _interpret_high_level(hl, models):
 
-    def interpret(text, data, threshold=.6):
-        if data['probability'] >= threshold:
```

Comment:
I removed this threshold because it didn't make much sense with models that had many classes - e.g. with 6 classes, 40% is much higher than the random baseline (16%)